components/nimbus/ios/Nimbus/NimbusApi.swift

```diff
@@ -202,16 +202,19 @@ public struct NimbusServerSettings {
 
 public let remoteSettingsCollection = "nimbus-mobile-experiments"
 
-/// Name and channel of the app, which should agree with what is specified in Experimenter.
+/// Name, channel and specific context of the app which should agree with what is specified in Experimenter.
+/// The specifc context is there to capture any context that the SDK doesn't need to be explictly aware of.
 ///
 public struct NimbusAppSettings {
-    public init(appName: String, channel: String) {
+    public init(appName: String, channel: String, appSpecificContext: [String: String]? = nil) {
         self.appName = appName
         self.channel = channel
+        self.appSpecificContext = appSpecificContext
     }
 
     public let appName: String
     public let channel: String
+    public let appSpecificContext: [String: String]?
```

Comment:
I'm not wild about the naming, although I prefer `appSpecificContext` to `specificContext`. FWIW [`Context` has a very specific meaning][1] in Android development.

I prefer `customAttributes` or `appSpecificTargetingAttributes`, or some such. Everyone likes custom things.

[1]: https://developer.android.com/reference/android/content/Context

I appreciate you've already renamed once, so please take this as optional.